docs/tempo/website/setup/set-up-test-app.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Set up a test app for a Tempo cluster
+menuTitle: Set up a test application for a Tempo cluster
+description: Learn how to set up a test app for your Tempo cluster and visualize data.
+weight: 400
+---
+
+# Set up a test application for a Tempo cluster
+
+Once you've set up a Grafana Tempo cluster, you need to write some traces to it and then query the traces from within Grafana.
+
+## Before you begin
+
+You'll need:
+
+* Grafana 9.0.0 or higher
+* Microservice deployments require the Tempo querier URL, for example: `http://query-frontend.tempo.svc.cluster.local:3200`
+
+Refer to [Deploy Grafana on Kubernetes](https://grafana.com/docs/grafana/latest/setup-grafana/installation/kubernetes/#deploy-grafana-on-kubernetes) if you are using Kubernetes.
+Otherwise, refer to [Install Grafana](https://grafana.com/docs/grafana/latest/installation/) for more information.
+
+## Set up remote-write to your Tempo cluster
```

Comment:
If you want to use the code method, you can use `remote_write` (an underscore). You can also just use **remote write**

Reply:
Good catch. Fixing that.